protos/sources/vega/vega.proto

```diff
@@ -1095,6 +1104,52 @@ message LiquidityProvision {
   string reference = 12;
 }
 
+// Liquidity provider commitment for spot market
+message SpotLiquidityProvision {
+  // Status of a liquidity provision order.
+  enum Status {
+    // Default value
```

Comment:
When would the default value be returned?

Reply:
I don't think it ever will, I changed the text to be always invalid (to match the rest of the definitions)